org-code-javabuilder/media/src/main/java/org/code/media/Pixel.java

```diff
@@ -1,23 +1,33 @@
 package org.code.media;
 
 public class Pixel {
+  private Image image;
+  private int x;
+  private int y;
```

Comment:
Good question--this is because curriculum wanted the concept of a pixel and they wanted it to be different from a `Color`. I hadn't thought about needing to update x, y and image on manipulation, but it's a good call-out. Will check with curriculum.

Reply:
Ah you can't swap pixels with each other, you can only set the Pixel within theater with `(x, y, color)`, so we shouldn't have to worry about those references changing